packages/host/src/Host.tsx

```diff
@@ -51,74 +175,80 @@ const Host: FC = () => {
         <IntlProvider locale={locale}>
           <AppThemeProvider>
             <BrowserRouter>
-              <SnackbarProvider maxSnack={3}>
-                <Viewport>
-                  <Box display="flex">
-                    <AppDrawer />
-                    <Box display="flex" flex={1} flexDirection="column">
-                      <AppBar />
-                      <Box flex={1} display="flex">
-                        <React.Suspense fallback={'Loading'}>
-                          <Routes>
-                            <Route
-                              path={RouteBuilder.create(AppRoute.Dashboard)
-                                .addWildCard()
-                                .build()}
-                              element={<DashboardService />}
-                            />
-                            <Route
-                              path={RouteBuilder.create(AppRoute.Customers)
-                                .addWildCard()
-                                .build()}
-                              element={<CustomerContainer />}
-                            />
-                            <Route
-                              path={RouteBuilder.create(AppRoute.Suppliers)
-                                .addWildCard()
-                                .build()}
-                              element={
-                                <Heading locale={locale}>suppliers</Heading>
-                              }
-                            />
-                            <Route
-                              path={RouteBuilder.create(AppRoute.Stock)
-                                .addWildCard()
-                                .build()}
-                              element={<Heading locale={locale}>stock</Heading>}
-                            />
-                            <Route
-                              path={RouteBuilder.create(AppRoute.Tools)
-                                .addWildCard()
-                                .build()}
-                              element={<Heading locale={locale}>tools</Heading>}
-                            />
-                            <Route
-                              path={RouteBuilder.create(AppRoute.Reports)
-                                .addWildCard()
-                                .build()}
-                              element={
-                                <Heading locale={locale}>reports</Heading>
-                              }
-                            />
-                            <Route
-                              path={RouteBuilder.create(AppRoute.Messages)
-                                .addWildCard()
-                                .build()}
-                              element={
-                                <Heading locale={locale}>messages</Heading>
-                              }
-                            />
-                            <Route path="/" element={<DashboardService />} />
+              <CommandK>
```

Comment:
more nesting than a bird sanctuary

Reply:
🤣   Heh, yeah.

I ended up going with a `DependencyProvider` in cold chain to handle it all..